src/components/PathwaysProvider.tsx

```diff
@@ -43,6 +44,23 @@ export const PathwaysProvider: FC<PathwaysProviderProps> = memo(function Pathway
     [setPathways]
   );
 
+  const addPathwayFromFile = useCallback(
+    (file: File) => {
+      const reader = new FileReader();
+      reader.onload = (event: ProgressEvent<FileReader>): void => {
+        if (event.target?.result) {
```

Comment:
Duplicated `if` here. Sometimes you just really want to make extra sure it's not null

Reply:
just wanted to... double check